creator-node/src/services/sync/processSync.js

```diff
@@ -353,7 +352,8 @@ async function processSync (serviceRegistry, walletPublicKeys, creatorNodeEndpoi
       let redisKey = redis.getNodeSyncRedisKey(wallet)
       await redisLock.removeLock(redisKey)
     }
-    logger.info(redisKey, `DURATION SYNC ${Date.now() - start}`)
+
+    logger.error(redisKey, `Sync complete for wallets: ${walletPublicKeys.join(',')}. Status: ${errorObj ? 'Error: ' + errorObj.message : 'Success'}. Duration sync: ${Date.now() - start}. From endpoint ${creatorNodeEndpoint}.`)
```

Comment:
prob should be `logger.error` vs `logger.info` on this same errorObj conditional right

Reply:
i think it's cleaner to have 2 messages and remove the ternary. i'll make that change

Reply:
sg